veritable-react-gui/src/components/AgentHolder/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js

```diff
@@ -36,7 +36,7 @@ export default function ColumnLeftWrap({ origin }) {
 
               {dataCredentialSets.length > 0 &&
                 dataCredentialSets.map((i, k) => (
-                  <div key={k} className="card mb-3">
+                  <div key={i.referent} className="card mb-3">
```

Comment:
There is also

```js
i.attrs.timestamp // E.g.: "1647429394"
```

but perhaps `referent` works better because this one is generated using UUID.